third_party/unix/README

```diff
@@ -0,0 +1 @@
+Package unix is copied from [golang/sys/internal-branch.go1.17-vendor](https://github.com/golang/sys/tree/internal-branch.go1.17-vendor) to avoid importing the whole golang sys repo.
```

Comment:
There are plenty much code being copied, does it really save anything compared to importing the repo?

Reply:
README updated. Previous README is not accurate. If I understand correctly, "importing the repo" means we need to create a new repo(with the patch of multicast syscall based on `golang/sys`) and import it. I think this is not a good way to do it.

Reply:
Is it possible to keep the newly added types in a separate file and import the original module without changing its code?